osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/utils/JavaTimeHelpers.java

```diff
@@ -34,4 +38,23 @@ public static int getOffsetForZonedDateTimeInMillis(final ZonedDateTime dateTime
   public static boolean isDayLightSavingsActive(final ZonedDateTime dateTime) {
     return dateTime.getZone().getRules().isDaylightSavings(dateTime.toInstant());
   }
+
+  public static ZonedDateTime shiftZoneToUTC(final ZonedDateTime dateTime) {
+    final int offset = dateTime.getZone().getRules().getOffset(Instant.now()).getTotalSeconds();
+    return dateTime.plusSeconds(offset).withZoneSameInstant(ZoneId.of("UTC"));
```

Comment:
Misschien is het makkelijker om een instant van de ZonedDateTime te maken en daan een nieuwe date time aan te maken.
Anders ga je ook het originele object muteren.

Reply:
Hier moet nog JavaDoc bij